### DISCUSSION TITLE
fuzzy skin outside surface only

### DISCUSSION THREAD

**Discussion by codegrue** [sn4k3/UVtools#877] [Q&A]
I have a concave (i.e. partially hollow) shaped object, essentially an electronics housing, and I would like to apply Pixel Arithmetic to just the outside surfaces and not affect the inside surfaces. The inside surfaces are dimensional, and I need them smooth to join other parts. Is there a way to configure this? I have been playing with it for some time and have not found a way.

**Comment by sn4k3** [comment]
Not possible, the best setting will apply to visible surface, even if trapped it will be considered "visible".
BUT there's a way to bypass, UVtools have so many tools that sometimes you just need to think what combination can get your result.

Apply the fuzzy, then with the ROI/select tool, select the area you want to amend, then use default pixel arithmetic settings to turn all pixels white on that area. This will work if you can select your area correctly without touch the other surface AND if your noise pattern does not set zeros AND most probably if you dont use AA. If set zeros you can run morph - Gap close with 2 iterations to try to smooth, and also can run blur with 1 sample to smooth it out.

![image](https://github.com/sn4k3/UVtools/assets/113281/7f14c477-b432-4d93-9e9e-42c56de5f922)

**Reply by codegrue** [reply]
Thank you so much for the guidance on this. Need to play with this workflow.
